examples/energy_wizard/run_app.py

```diff
@@ -61,7 +62,16 @@ def get_corpus():
 @st.cache_resource
 def get_wizard():
     """Get the energy wizard object."""
-    corpus = get_corpus()
+
+    # Getting Corpus of data. If no corpus throw error for user.
+    try:
+        corpus = get_corpus()
+    except Exception:
+        print("Error: Have you run 'retrieve_docs.py'?")
```

Comment:
@jordantcox alternatives to print() would be raising a custom exception but because this is streamlit weirdness i think this makes sense 👍